helm/platform-health/values.yaml

```
@@ -0,0 +1,69 @@
+---
+# Common values
+common_labels: ~
+common_annotations: ~
+
+# Application container :: custom configuration
+image:
+  repository: platform-health
+  tag: latest
+  pullPolicy: Always
+
+# Container :: custom configuration
+container:
+  port: &port 8080
+  args:
+    - -vv # verbose mode
+
+# Service/LoadBalancer :: custom configuration
+loadBalancer:
+  enabled: true
+  port: *port
+
+# Deployment :: custom configuration
+deployment: ~
+
+# ServiceAccount :: custom configuration
+serviceAccount: ~
+
+# RoleBinding :: custom configuration
+roleBinding: ~
+
+# Role :: custom configuration
+role:
+  # Allows the application to read the Secrets
+  enableSecrets: true
```

```suggestion
  enableSecrets: false
```
I don't like charts that default to giving themselves access to all my Secrets without making that crystal clear upfront.